api/apps/api/src/modules/specification/adapters/specification-feature.api.entity.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { SpecificationFeatureConfigApiEntity } from './specification-feature-config.api.entity';
+
+@Entity(`specification_feature`)
```

Comment:
the kept convention is to use plurals

Reply:
Since this is a full entity now, we will also need an `order` (or something like that) column to keep the ordering of features in a specification consistent with what API clients send (or the frontend to start with may see features jumping around depending on contingent factors such as order of insert/update in db)

Reply:
@hotzevzl sorting on serializer/controller level should be enough, shouldn't it?

if it is needed at all (https://vizzuality.slack.com/archives/C01G7PRQB96/p1628593316134400)

Reply:
ok, so if we are echoing back (with any augmentation) the specification as received in the request payload then we don't need to keep an order of the children of `specification_features` as stored in db.

if we are composing the response from the data we have in db, then keeping the same order of geofeatures as provided in the request payload would be preferable for a consistent user experience - we could sort at serializer level, but without other order hints we could only order alphabetically here, unless I am misunderstanding what you suggest.